process_data.sh

```diff
@@ -223,6 +209,10 @@ if [[ -e "${file_t1w}.nii.gz" && -e "${file_mton}.nii.gz" && -e "${file_mtoff}.n
   # Register MT->T1w
   sct_register_multimodal -i ${file_mton}.nii.gz -d ${file_t1w}.nii.gz -dseg ${file_t1w_seg}.nii.gz -param step=1,type=im,algo=rigid,slicewise=1,metric=CC -x spline -qc ${PATH_QC} -qc-subject ${SUBJECT}
   file_mton="${file_mton}_reg"
+  # Copy json files to match file basename (it will later be used by sct_compute_mtsat)
+  cp ${SUBJECT}_acq-T1w_MTS.json ${file_t1w}.json
+  cp ${SUBJECT}_acq-MToff_MTS.json ${file_mtoff}.json
+  cp ${SUBJECT}_acq-MTon_MTS.json ${file_mton}.json
```

Comment:
Are these lines really necessary? It looks that e.g. ${file_t1w} variable contains exactly the same filename how the original json file:

https://github.com/spine-generic/spine-generic/blob/1c113421b4006dc65fc067ebdd38eb6344c87631/process_data.sh#L192-L194

Reply:
yes, because file names get updated, eg here:
https://github.com/spine-generic/spine-generic/blob/1c113421b4006dc65fc067ebdd38eb6344c87631/process_data.sh#L204

Reply:
yes, because those files are processed, new .nii.gz files are generated, but not new json files, E.g.: 
https://github.com/spine-generic/spine-generic/blob/1c113421b4006dc65fc067ebdd38eb6344c87631/process_data.sh#L204